docs/faq.md

```diff
@@ -54,7 +54,7 @@ To manually fix field or table names if they still look wrong, you can go to the
 
 ### Can I add headings, free text, section dividers, or images to my dashboards?
 
-Yes, by using [text cards](users-guide/07-dashboards.html#adding-headings-or-descriptions-with-text-cards).
+Yes, by using [text cards](users-guide/07-dashboards.md#adding-headings-or-descriptions-with-text-cards).
```

Comment:
This is wrong; it should be `.html` in cases where we're using anchor/hash links. By some odd quirk of a doc build process, these kinds of links don't properly get `.md` converted to `.html`, which caused a bunch of broken links I had to fix manually.

Reply:
This was a conflict that I resolved incorrectly. I'll fix it.